TYPO3.Media/Classes/TYPO3/Media/Command/MediaCommandController.php

```diff
@@ -123,6 +175,31 @@ public function clearThumbnailsCommand()
     }
 
     /**
+     * Generate uninitialized asynchronous thumbnails
+     *
+     * @param integer $limit
+     * @return void
+     */
+    public function generateThumbnailsCommand($limit = null)
```

Comment:
okay, sorry, I'm reading the code from top to bottom, so didn't see that there's another command called "generateThumbnails". I don't know yet about presets, so by now - when I just read the user facing API - it's a bit confusing, what means what.